tutorials/users/users-intro.md

```diff
@@ -0,0 +1,26 @@
+# Preamble
+
+This section covers the user side of things on Marble.
+
+::::{grid} 1 1 2 3
+:class-container: text-center
+:gutter: 3
+
+:::{grid-item-card}
+:link: marble/getting-started
+:link-type: doc
+:class-header: sd-bg-dark, sd-text-light, sd-font-weight-bold
```

Comment:
Let's be consistent with the classes here. If we remove the ones in intro.md we should also remove them here